features/step_definitions/sample_file_steps.rb

```diff
@@ -53,6 +53,10 @@ def a
   contents = <<-EOS.strip_heredoc
     # clean class for testing purposes
     class Clean
+      def initialize(sub)
+        @sub = sub
+      end
+
```

Comment:
Wait, couldn't we rather improve the code in our "clean class" to not have instance variables at all?
Why not just use "samples/clean.rb" here?